.github/workflows/build_and_push.yaml

```diff
@@ -45,4 +45,4 @@ jobs:
           cache-to: type=inline
           tags: |
             lemonsaurus/blackbox:latest
```

Comment:
I think the `latest` tag should only refer to the latest _release version_, not the latest commit. That is the normal convention.

So, let's tag with `latest` when we do the semver release, and then here we can tag with `main` instead.

Reply:
Reasonable. Fixed.